alchemy/src/cloudflare/bundle/plugin-node-compat.ts

```diff
@@ -1,10 +1,13 @@
 import type { Plugin } from "esbuild";
 import type { NodeJSCompatMode } from "miniflare";
+import { builtinModules } from "node:module";
 import { relative } from "node:path";
 import chalk from "picocolors";
 import { dedent } from "../../util/dedent.ts";
 import { logger } from "../../util/logger.ts";
 
+const NODEJS_MODULES_RE = new RegExp(`^(node:)?(${builtinModules.join("|")})$`);
```

Comment:
By the way, this can have differing behavior on `bun` because `builtinModules` is different. Not sure what I had before, but worth checking we didn't regress anything there.

Reply:
good catch. added that back